schemas/com.snowplowanalytics.snowplow/media_player/jsonschema/1-0-0

```diff
@@ -0,0 +1,76 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Common Context Schema for a media player event",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "currentTime": {
+      "type": "number",
+      "description": "The current playback time",
+      "minimum": 0,
+      "maximum": 9007199254740991
+    },
+    "duration": {
+      "type": [
+        "number",
+        "null"
```

Comment:
Yeah, it's reported as `+Infinity` for a stream by JavaScript, it does feel like it might be missing a bit of context in the case of a live video by leaving it as null now that you mention it. Perhaps a `isLive` or `isStream` bool property could be added to be more explicit?

Reply:
`duration: null, isStream: true` is a nice solution (assuming duration being null is the only property you need to look at to know it's a stream).

Reply:
'isLive' has been added